components/common/accordion/accordion.scss

```diff
@@ -0,0 +1,61 @@
+.Accordion {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    .accordion-item {
+        border: 1px solid #ccc;
```

Comment:
nit: can we use a variable here to have support for a future theming approach (if we ever decide to implement that)?